src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentThreadPartial.cshtml

```diff
@@ -89,15 +96,48 @@
                     <div class="m-2 d-flex justify-content-between">
                         @if (Model.IsResolved)
                         {
-                            <form data-post-update="comments" method="post" asp-controller="Comments" asp-action="Unresolve" asp-route-reviewId="@Model.ReviewId">
-                                <button type="submit" name="submit" value="Submit" class="btn btn-outline-dark">Unresolve</button>
-                            </form>
+                            @if(Model.Comments.First().ResolutionLocked && Model.Comments.First().Username != User.GetGitHubLogin())
+                            {
+                                <span asp-resource="@Model" asp-requirement="@ApproverRequirement.Instance">
+                                    <form data-post-update="comments" method="post" asp-controller="Comments" asp-action="Unresolve" asp-route-reviewId="@Model.ReviewId" asp-requirement="@ApproverRequirement.Instance">
+                                        <button type="submit" name="submit" value="Submit" class="btn btn-outline-dark">Unresolve</button>
+                                    </form>
+                                </span>
+                                <p>
+                                    &nbsp;&nbsp;Please ask
+                                    <a username="@Model.Comments.First().Username" class="font-weight-bolder text-dark mt-0">@Model.Comments.First().Username</a>
```

Comment:
I don't think this is correct. Resolve button will only be visible to Approver. If first comment is created by non-approver then this message is incorrect. So a better solution is to use a different asp-requirement that check whether user is approver or the username in first comment instead of using ApproverRequirement. Implementation will be simialr to ApproverRequirement and ApproverRequirementHandler with an additional check verify the comment's user if user is not an approver.